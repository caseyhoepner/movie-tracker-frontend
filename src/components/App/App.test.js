import React from 'react';
import { shallow } from 'enzyme';
import { App } from './index.js';

describe('App', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})