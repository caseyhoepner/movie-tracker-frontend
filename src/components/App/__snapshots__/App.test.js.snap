// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<div
  className="App"
>
  <header>
    <div
      className="logo"
    >
      <img
        alt="An icon of Wes Anderson's face"
        className="logo-icon"
        src="wes-logo.png"
      />
      <h1>
        WesTracker
      </h1>
    </div>
    <Nav />
  </header>
  <Route />
</div>
`;
